VWOSdk/Contract/IVWOClient.cs

```diff
@@ -25,61 +27,72 @@ public interface IVWOClient
         /// Tracks a conversion event for a particular user for a running server-side campaign.
         /// </summary>
         /// <param name="campaignTestKey">Campaign key to uniquely identify a server-side campaign.</param>
         /// <param name="userId">User ID which uniquely identifies each user.</param>
         /// <param name="goalIdentifier">The Goal key to uniquely identify a goal of a server-side campaign.</param>
-        /// <param name="revenueValue">The Revenue to be tracked for a revenue-type goal.</param>
+        /// <param name="options">Dictionary for passing extra parameters to activate</param>
         /// <returns>
         /// A boolean value based on whether the impression was made to the VWO server.
         /// True, if an impression event is successfully being made to the VWO server for report generation.
         /// False, If userId provided is not part of campaign or when unexpected error comes and no impression call is made to the VWO server.
         /// </returns>
-        bool Track(string campaignTestKey, string userId, string goalIdentifier, string revenueValue = null);
-
+        bool Track(string campaignTestKey, string userId, string goalIdentifier, Dictionary<string, dynamic> options = null);
         /// <summary>
-        /// Tracks a conversion event for a particular user for a running server-side campaign.
+        /// Identifies whether the user becomes a part of feature rollout/test or not.
         /// </summary>
         /// <param name="campaignTestKey">Campaign key to uniquely identify a server-side campaign.</param>
         /// <param name="userId">User ID which uniquely identifies each user.</param>
-        /// <param name="goalIdentifier">The Goal key to uniquely identify a goal of a server-side campaign.</param>
-        /// <param name="revenueValue">The Revenue to be tracked for a revenue-type goal.</param>
+        /// <param name="options">Dictionary for passing extra parameters to activate</param>
         /// <returns>
-        /// A boolean value based on whether the impression was made to the VWO server.
+        /// /// A boolean value based on whether the impression was made to the VWO server.
         /// True, if an impression event is successfully being made to the VWO server for report generation.
         /// False, If userId provided is not part of campaign or when unexpected error comes and no impression call is made to the VWO server.
         /// </returns>
-        bool Track(string campaignTestKey, string userId, string goalIdentifier, int revenueValue);
+         bool IsFeatureEnabled(string campaignTestKey, string userId, Dictionary<string, dynamic> options = null);
 
         /// <summary>
-        /// Tracks a conversion event for a particular user for a running server-side campaign.
+        /// Returns the feature variable corresponding to the variable_key passed. It typecasts the value to the corresponding value type found in settings_file

```

Comment:
no snake_case anywhere in .NET please, `variable_key` Python ported comments need to be checked.

Reply:
ok